lib/inquirer-prompt.js

```diff
@@ -15,38 +15,16 @@ const chalk = require('chalk');
 
 module.exports = function prompt(questions, config) {
 	inquirer.prompt(questions).ui.process
-	.reduce(function (newOpts, ans) {
-		return attachAnswers(newOpts, ans);
-	}, Object.assign(config, require('./utils/initial-config')))
-	.flatMap(newOpts => {
-		const parser = require('./parser/index');
-		return parser(null, checkEmptyAnswers(newOpts));
-	})
-	.subscribeOnError( (err) => process.stdout.write(err.toString()));
-};
-
-/*
-* @function attachAnswers
-*
-* Adds the answers from the inquirer instance if the type of the question is
-* equal to the configuration property
-*
-* @param { Object } config - initial questions provided from --init
-* @param { Object } answers - The answers passed from the inquirer instance
-* @returns { Object } newConfig - An new object with the answers added to it
-*/
+	.subscribe(ans => {
+		config = require(`./init-transforms/${ans.name}`)(config, ans.answer);
```

Comment:
If `config` here is an instance of webpack-chain, then returning it here won't do anything since all interactions with the API all build up an internal configuration object. Which means:

```js
const cached = config;
config = require(`./init-transforms/${ans.name}`)(config, ans.answer);

assert(config === cached);
```

Returning it won't change what you've passed in, unless you return a different webpack-chain instance, which isn't happening here. It could technically just be:

```js
require(`./init-transforms/${ans.name}`)(config, ans.answer);
```